torch/csrc/jit/script/module.cpp

```diff
@@ -24,7 +24,7 @@ static FunctionSchema defaultSchemaFor(Method& method) {
   for(size_t i = 0; i < g.outputs().size(); ++i) {
     returns.push_back({"", DynamicType::get()});
   }
-  return { method.name(), std::move(args), std::move(returns) };
+  return { "", std::move(args), std::move(returns) };
```

Comment:
Why do we no longer use method.name() here?

Reply:
Forgot to revert that.